app/assets/stylesheets/components/page_specific/_category_link_list.scss

```diff
@@ -27,6 +27,10 @@
     // This is to override the highly specific .editorial list styling
     content: none !important;
   }
+
+  > .category-link-list__item__link {
```

Comment:
Why is this nested in here and not in the declaration below - `.category-link-list__item__link` - specificity?